Pi_java_stuff/runtimeDir/Selection.properties

```diff
@@ -0,0 +1,6 @@
+sauce1=Nutella
+sauce2=Marmelada
+sauce3=�alca Dreka
+sauce1Img=http://www.nutella.com/nutella-gl-theme/images/custom/Nutella.png
+sauce2Img=http://streamlinefoods.co.uk/wp-content/uploads/2014/02/Raspberry-Seedless-Jam_Streamline.png
```

Comment:
a si siguren, da rabs sploh properties file za 5vrstic?

On 30.5.2016 8:34, grizeldi wrote:
>
> In Pi_java_stuff/runtimeDir/Selection.properties 
> <https://github.com/mucamaca/vic-palacinke/pull/29#discussion_r65029151>:
>
> > @@ -0,0 +1,6 @@
> > +sauce1=Nutella
> > +sauce2=Marmelada
> > +sauce3=�alca Dreka
> > +sauce1Img=http://www.nutella.com/nutella-gl-theme/images/custom/Nutella.png
> > +sauce2Img=http://streamlinefoods.co.uk/wp-content/uploads/2014/02/Raspberry-Seedless-Jam_Streamline.png
> Ja vem, sj sm probavu ampak ni hotu odpret lokalnga patha. Mogoče če 
> probam z "file://path/to/image".
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/mucamaca/vic-palacinke/pull/29/files/8683d142345862d232f647833808fd70ef369b46#r65029151>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AO7w9U4uG4pmXpMARPRBC9WAwHPXdw7kks5qGoT8gaJpZM4IpJO3>.
>